src/io.jl

```diff
@@ -99,8 +99,14 @@ function Base.write(io::IO, d::MRCData; compress = :none)
     T = datatype(h)
     data = parent(d)
     fswap = bswapfromh(h.machst)
-    for i in eachindex(data)
-        @inbounds sz += write(newio, fswap(T(data[i])))
+    unit_vsize = 4096 ÷ sizeof(T)
```

Comment:
I prefer to be explicit regarding `div`
```suggestion
    unit_vsize = div(4096, sizeof(T))
```